### DISCUSSION TITLE
With typescript getting form$.value is possibly 'null'

### DISCUSSION THREAD

**Discussion by tboerger** [vueform/vueform#342] [Questions]
Hey, I have followed all the guides to define my form with `ref="form$"` and I have also defined `const form$ = ref(null);` within my typescript vue code. Beside that I'm calling `form$.value.update(myData);` to load content into the form when editing a record.

When I try to build this code I'm always getting the error `TS18047: 'form$.value' is possibly 'null'.` and I'm not sure how to properly solve that? I haven't been able to find an answer to that within the docs.

**Comment by adamberecz** [comment]
Would importing [Vueform](https://github.com/vueform/vueform/blob/main/types/index.d.ts#L3870) from type definitions and assigning it to `form$` ref work?
